zulipterminal/model.py

```diff
@@ -654,6 +654,16 @@ def _handle_typing_event(self, event: Event) -> None:
                 else:
                     raise RuntimeError("Unknown typing event operation")
 
+    def are_valid_recipients(self, recipient_emails: List[str]) -> bool:
+        user_emails = {user['email'] for user in self.users}
```

Comment:
This seems potentially costly. Unless I'm mistaken, we already have this data in `self.user_dict[email]` through subscription.

Reply:
Good point :+1: